lib/lcdbwf/R/results.R

```diff
@@ -92,7 +92,5 @@ build_results_tabs <- function(res_list, dds_list, config, text){
 #' @return Boolean TRUE if results object's pvalues were provided by
 #' nBinomLRT and FALSE if the Wald test was used.
 check_LRT <- function(res_i) {
-  mcols_pval <- mcols(res_i)$description[9]
-  mcols_pval <- grepl('LRT', mcols_pval)
-  return(mcols_pval)
+  mcols <- grepl('LRT', mcols(res_i)$description[9])
```

Comment:
does this need `S4Vectors::`? Also seems to be overwriting `mcols` (albeit within the function scope, but still could be confusing).